snu-ece-bsc-thesis.cls

```diff
@@ -82,10 +92,17 @@
     \bicaptionsetup[table]{name=표}{name=Table}
     \bicaptionsetup[figure]{name=그림}{name=Figure}
 
-    \let\abstractname\l_snu_ece_bsc_thesis_abstractname_ko
+    \let\abstractname\l_snu_ece_bsc_thesis_abstractstyle_ko
     \let\contentsname\l_snu_ece_bsc_thesis_contentsname_ko
     \let\listtablename\l_snu_ece_bsc_thesis_listtablename_ko
     \let\listfigurename\l_snu_ece_bsc_thesis_listfigurename_ko
+
+    % Title and ToC Formatting
+    \RequirePackage{titlesec}
+
+    \titlespacing*{\chapter}{0pt}{-10mm}{10mm}
+    \titleformat{\chapter}[block]{\centering \huge \bfseries}%
```

Comment:
No trailing % required in an expl3 context.

Reply:
```diff
@@ -76,7 +76,7 @@
     \RequirePackage{titlesec}

     \titlespacing*{\chapter}{0pt}{-10mm}{10mm}
-    \titleformat{\chapter}[block]{\centering \huge \bfseries}%
+    \titleformat{\chapter}[block]{\centering \huge \bfseries}
                 {Chapter\ \thechapter}{22pt}{}
   }
   { % Korean main
@@ -101,7 +101,7 @@
     \RequirePackage{titlesec}

     \titlespacing*{\chapter}{0pt}{-10mm}{10mm}
-    \titleformat{\chapter}[block]{\centering \huge \bfseries}%
+    \titleformat{\chapter}[block]{\centering \huge \bfseries}
                 {제\ \thechapter\ 장}{20pt}{}
   }
```